cli/.js/src/main/scala/aqua/ArgOpts.scala

```diff
@@ -0,0 +1,143 @@
+package aqua
+
+import aqua.builder.ArgumentGetter
+import aqua.raw.value.{LiteralRaw, ValueRaw, VarRaw}
+import cats.data.{NonEmptyChain, NonEmptyList, Validated, ValidatedNec, ValidatedNel}
+import Validated.{invalid, invalidNec, valid, validNec, validNel}
+import aqua.parser.expr.func.CallArrowExpr
+import aqua.parser.lexer.{Literal, VarLambda}
+import aqua.parser.lift.Span
+import aqua.types.BottomType
+import cats.{~>, Id}
+import cats.effect.Concurrent
+import com.monovore.decline.Opts
+import cats.syntax.applicative.*
+import cats.syntax.apply.*
+import cats.syntax.flatMap.*
+import cats.syntax.functor.*
+import cats.syntax.traverse.*
+import fs2.io.file.Files
+
+import scala.scalajs.js
+import scala.scalajs.js.JSON
+
+case class FuncWithData(name: String, args: List[ValueRaw], getters: Map[String, ArgumentGetter])
+
+object ArgOpts {
+
+  def spanToId: Span.S ~> Id = new (Span.S ~> Id) {
+
+    override def apply[A](span: Span.S[A]): Id[A] = {
+      span._2
+    }
+  }
+
+  // Parses a function name and arguments from a string
+  def funcOpt: Opts[(String, List[ValueRaw])] =
+    Opts
+      .option[String]("func", "Function to call with args", "f")
+      .mapValidated { str =>
+        CallArrowExpr.funcOnly.parseAll(str) match {
+          case Right(exprSpan) =>
+            val expr = exprSpan.mapK(spanToId)
+
+            val args = expr.args.collect {
+              case Literal(value, ts) =>
+                LiteralRaw(value, ts)
+              case VarLambda(name, _) =>
+                // TODO why BottomType?
+                VarRaw(name.value, BottomType)
```

Comment:
Why BottomType? Because you skip the semantics?

Reply:
cause i don't know what types of these arguments

Reply:
Why?

Reply:
#435

Reply:
Plz add this reference to this TODO in the code, so that we don't miss it